nrfs/README.md

```diff
@@ -0,0 +1,33 @@
+# nRF Services
+
+## Overview
+
+NRFS (nRF Services) is a library used for accessing services provided by the System Controller core.
+The number of available services may vary depending on your device. Check the product specification
+to see what services are available for the specific chip.
+
+## Supported SoCs and SiPs
+
+* nRF54H20
+* nRF9230
+* nRF7140
+
+## Directories
+
+```
+
+nrfs
+├── helpers             # helper files, need to be shared between LDs and System Controller
```

Comment:
"LDs" is not described anywhere, does it mean "Local Domains"?

Reply:
yes, I'll remove this abbreviation